api/lambdas/withings/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "withings",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"No test specified\""
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "aws-sdk": "^2.1248.0",
+    "axios": "^1.2.1"
```

Comment:
NABD, but it would be nice to use a newer version as we're trying to keep all using the same... `packages` is on `1.3.5` for CW stuff.